src/test_pages/TimelinePage.ts

```diff
@@ -4,13 +4,16 @@ import { BasePage } from './BasePage'
 export default class TimelinePage extends BasePage {
   private date: Locator
   private operator: Locator
-  private operators_dropdown: Locator
+  public operators_dropdown: Locator
```

Comment:
Hi Eyal! Why do you change locators to public? It means that you can change the locator whenever you want, but this behavior is not so welcomed. Because the locators are something permanent.

Reply:
perhaps it should be changed to a getter

Reply:
Actually, there are getters in the class. Each locator should be threated by some getter.

Reply:
@DanLeshinsky I mean
```
public get operators_dropdown() {
    return this.page.locator("button[aria-label='Open']");
}
```

Reply:
Just to clarify what I was talking about. The locators declared in the class and then initialized in the constructor. According to getters, I mean that in each page class we usually add the functions as click, fill, scroll_to, clear etc. that uses the locators and inside you can access to them. And of course, you can create getters but there's no need for it, because you just need to create the function that can access to these locators.